src/local/update.go

```diff
@@ -9,46 +9,46 @@ import (
 func (m *Manager) UpdatePaths(state model.ObjectState, rename bool) {
 	object := state.LocalOrRemoteState()
 
-	switch v := state.(type) {
-	case *model.BranchState:
-		branch := object.(*model.Branch)
-
-		// Set paths
-		if v.Path == "" || rename {
-			v.Path = m.Naming().BranchPath(branch, branch.IsDefault)
-		}
-		v.ResolveParentPath()
-	case *model.ConfigState:
-		config := object.(*model.Config)
-
-		// Get parent - branch
-		branchKey := v.BranchKey()
-		branchManifest, found := m.manifest.GetRecord(branchKey)
-		if !found {
-			panic(fmt.Errorf("branch manifest wit key \"%s\" not found", branchKey))
+	// Update parent path
```

Comment:
Spolu s tymto problemom som musel vyriesit to, ze doteraz sme neriesili, ze cesta k objektu na disku musi byt unikatna.

V `naming.go` sa proste zobrala sablona a vygeneroval sa nazov adresara.

No ak maju napr. konfiguracie rovnake `name` a v sablone pre naming nebude ID, tak nastane kolizia.
Rovnako ako ked mame viacero config rows, co nemaju nastavene meno.

Preto som musel upravit sposob, ako sa cesty generuju.